yb-voyager/cmd/analyzeSchema.go

```diff
@@ -452,6 +454,25 @@ func checkDDL(sqlInfoArr []sqlInfo, fpath string) {
 			reportCase(fpath, "LANGUAGE C not supported yet.",
 				"", "", "FUNCTION", tbl[2], sqlInfo.formattedStmt)
 			summaryMap["FUNCTION"].invalidCount++
+		} else if regMatch := partitionColumnsRegex.FindStringSubmatch(sqlInfo.stmt); regMatch !=nil {
+			allColumns := strings.Trim(regMatch[3], `()`)
+			allColumns = strings.Trim(allColumns, " ")
+			allColumnsList := strings.Split(allColumns, "(")
+
+			primaryKeyColumns := allColumnsList[len(allColumnsList)-1]
+			primaryKeyColumns = strings.Trim(primaryKeyColumns, `()`)
+			partitionColumns := strings.Trim(regMatch[4], `()`)
+
+			partitionColumnsList := utils.CsvStringToSlice(partitionColumns)
+			primaryKeyColumnsList := utils.CsvStringToSlice(primaryKeyColumns)
+			for _, eachPartitionColumn := range partitionColumnsList {
+				eachPartitionColumn = strings.Trim(eachPartitionColumn, " ")
```

Comment:
`utils.CsvStringToSlice` cleans up whitespaces while converting to slice; not sure if this line is needed